docs/data-model.md

```diff
@@ -0,0 +1,29 @@
+# alerts_concierge Data Model
```

Comment:
suggestion: Maybe it's worth linking to this from the README for added visibility?

Reply:
Linked.